pages/teaminfo.js

```diff
@@ -5,23 +5,20 @@ import amber from 'public/images/amber.webp';
 import chris from 'public/images/chris.webp';
 import nasa from 'public/images/nasa.webp';
 import Layout from './components/layout';
-import { getSession } from "next-auth/react";
-import { readUser } from "./api/user";
+import { getUserFromSession } from "@/pages/api/user";
 
 export async function getServerSideProps(ctx) {
-  const session = await getSession(ctx);
-  if (session == null) {
+  const user = await getUserFromSession(ctx);
+  if (user == null) {
```

Comment:
what is the default value of props.user, isn't it null already?

Reply:
Yep, good catch!